### PR TITLE
Set up Rails

### PR DESCRIPTION
**Pull request by rlcheng**
https://www.pivotaltracker.com/story/show/116834213

Project setup with Ruby 2.3 and Rails 5.0 beta3 with install instructions in Readme

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  puts "== Installing dependencies =="
+  system "gem install bundler --conservative"
+  system "bundle check || bundle install"
+
+  # puts "\n== Copying sample files =="
+  # unless File.exist?("config/database.yml")
+  #   system "cp config/database.yml.sample config/database.yml"
+  # end
+
+  puts "\n== Preparing database =="
```
Comment (kkchu791) [inline bin/setup]: I think currently we don't want any db code

File: config/initializers/backtrace_silencers.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Be sure to restart your server when you modify this file.
```
Comment (kkchu791) [inline config/initializers/backtrace_silencers.rb]: unused file

File: config/initializers/cookies_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Be sure to restart your server when you modify this file.
+
+Rails.application.config.action_dispatch.cookies_serializer = :json
```
Comment (kkchu791) [inline config/initializers/cookies_serializer.rb]: json api doesn't need cookies

File: config/initializers/inflections.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
```
Comment (kkchu791) [inline config/initializers/inflections.rb]: unused file

File: config/initializers/mime_types.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
```
Comment (kkchu791) [inline config/initializers/mime_types.rb]: unused file

File: db/seeds.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```
Comment (kkchu791) [inline db/seeds.rb]: no need for seed file currently

File: Gemfile [hunk 7]
```diff
@@ -0,0 +1,15 @@
+source 'https://rubygems.org'
+
+ruby "2.3.0"
+
+gem 'rails', github: 'rails/rails'
+gem "active_model_serializers", github: "rails-api/active_model_serializers"
```
Comment (kkchu791) [inline Gemfile]: should we wait for later tickets to add this gem?

File: Gemfile [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+gem 'rspec_api_documentation'
```
Comment (kkchu791) [inline Gemfile]: there's another ticket for this adding this gem

File: Gemfile [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+group :development, :test do
+  gem 'byebug'
+  gem 'spring'
+end
+
+group :production do
```
Comment (kkchu791) [inline Gemfile]: no plans for production at the moment

**Comment by kkchu791** [comment]
:+1:  Looks good to me!

**Comment by thomas-jung** [comment]
I think this looks good :+1:

**Pull request merged**